output.hpp

```diff
@@ -0,0 +1,20 @@
+#pragma once
+#include<iostream>
+#include<fstream>
+#include<vector>
+#include<iomanip>
+using namespace std;
```

Comment:
You should remove this line since it will create namespace conflicts with the main.